src/Core/Label.js

```diff
@@ -67,6 +67,7 @@ class Label extends THREE.Object3D {
         });
 
         this.isLabel = true;
+        this.forceHidden = false;
```

Comment:
if layer.visible == false
then in `LabelLayer`, traverse all `labels/node.dom` to set hidden. 
or there is a main div to hide all labels?

Reply:
I wanted to do this at first, but we don't want to hide all labels, if there are multiple layers with labels in it. So each label needs to be treated individually.

Reply:
Could add a div for each label layer?

Reply:
Yes of course that is the perfect solution, sorry for missing this ! Should we still keep something like `forceHidden` though, as it will prevent from doing too much computation in the render loop ? Or maybe add a `layer` property to the label, so we can perform a `label.layer.visible` check in the renderer ?

Reply:
Or think about browsing the div tree for label rendering?

Reply:
I tried to find a solution using the `div` tree, but I can't see how it could be possible, besides having a list of all displayed labels, or something like that as reference. I updated the PR with the removal of `forceHidden` and a `div` for each `LabelLayer`.